src/Symfony/Component/DependencyInjection/Tests/Config/ContainerParametersResourceCheckerTest.php

```diff
@@ -62,11 +62,11 @@ public function isFreshProvider()
         }, false];
 
         yield 'fresh on every identical parameters' => [function (MockObject $container) {
-            $container->expects($this->exactly(2))->method('hasParameter')->willReturn(true);
-            $container->expects($this->exactly(2))->method('getParameter')
+            $container->expects(self::exactly(2))->method('hasParameter')->willReturn(true);
+            $container->expects(self::exactly(2))->method('getParameter')
```

Comment:
All these methods are defined as static anyway, shall we fix them through the whole codebase starting from 5.4 @nicolas-grekas ?

Reply:
No need to. That wouldn't provide any real added value but would consumer a lot of time for sure.